source/style-guide/style/version.txt

```diff
@@ -0,0 +1,94 @@
+.. _version-reference:
+
+==================
+Version Reference
+==================
+
+.. default-domain:: mongodb
+
+Documentation of features often requires reference to a specific or at least
+major release of the product to indicate when the feature became available,
+when the feature was changed, and when it was deprecated.
+
+Readers find this information useful as it can explain unexpected behavior in
+cases where they attempt to perform an action that is not possible on their
+specific release of the product.
+
+With the exception of release notes and changelogs, which serve as a historical
+document, we should not reference releases of products that we no longer support.
+That is, when a release of MongoDB Server reaches EOL,  references to this
+release should be removed.
+
+* Users don't need to know a feature was introduced in a release that is EOL,
+  since it suggests a text that is out of date.
+
+* Users don't need to know about features that were removed in a release 
+  that is EOL, since it means the feature is unavailable in all supported 
+  releases.
+
+* Users don't need to know about features that were changed in a release
+  that is EOL, since the change now applies to all available releases.
+
+This topic provides guidelines on how to reference product versions in order to
+make it easier to find these references once the release reaches EOL.
```

Comment:
```suggestion
Use the following guidelines to reference product versions so that
you can more easily find these references when the release reaches EOL.
```